### ISSUE TITLE
[Channel-Permissions] Show if they are synced with category perms or not

### ISSUE THREAD

**Issue by Balls0fSteel** [Farcrada/DiscordPlugins#49]
### Is your request related to a problem? Please describe.

I love the plugin and it makes my job as a mod/admin a LOT easier. However, thanks to Discord, it's a real pain to see if a channel is in sync with a category or not. In fact, it's pretty much impossible to see off the bat.

You have to head into **each** channel settings, and see if they are in sync.
Not sync? Press sync. Did you press the button? I don't know. You don't either.
Time to check back again. Now do this for another 30 channels.
Did you do it on all of them? I don't know. You don't either.
Oh okay, you've spent 5 hours doing this. It's 3 days later. But are you SURE you have done this on all channels?

Do you see the issue?
Double-checking the perm sync thing is a major problem. 
Could you fix this another Discord usability blunder?

Thank you for your work!

Ps.: Thanks to this messy feature my server already got spammed by some NSFW imagery (that Discord conveniently allowed through regardless of server & user safety settings...)

### Describe the solution you'd like

- Add a short message, such as "Synced with: WELCOME"
- Maybe add a colored/uncolored icon, like a Firefox Sync icon (you know, two arrows, showing sync). 
IF in sync, colored icon.
IF NOT in sync, gray/dark icon.

**Comment by Farcrada** [comment]
I love it. I'll get on this.

**Comment by Farcrada** [comment]
I can imagine you've been waiting on this. Would just like to let you know I'm working on it and slowly getting a working concept together.

**Comment by Balls0fSteel** [comment]
Oh, thank you very much. I really appreciate your work (fixing what Discord should have added from the get-go). Thank you!

**Comment by Farcrada** [comment]
Should be added now. Enjoy!

**Comment by Balls0fSteel** [comment]
Thank you so much. 
- Unfortunately, it doesn't want to work after the update - this seems to happen random times. It was working just before though. Probably worth its own ticket though, don't want to mess up your tracker.
- The Plugin Repo (which is a plugin itself) downloads a pretty old plugin that just crashes Discord all the time lol.

**Comment by Farcrada** [comment]
I had hoped I was fast enough: https://github.com/Farcrada/DiscordPlugins/commit/38a8b45490699e351b477ea0b69891c80e0979fb Haha! Unfortunate.

I will mention there was a recent Discord update which broke the library which might've contributed: https://github.com/rauenzi/BDPluginLibrary/pull/94#issue-715024611
